### PR TITLE
Swig Priors

### PR DESCRIPTION
**Pull request by robbietuk**
Allows for the Quadratical and Parallel Level Sets prior objects to be called from MatLab and Python.

I have used this when investigating values of a prior throughout reconstruction.

### CODE REVIEW

File: src/swig/stir.i [hunk 1]
```diff
@@ -1668,3 +1668,11 @@ namespace stir {
   stir::RegisteredParsingObject<stir::BackProjectorByBinUsingProjMatrixByBin,
      stir::BackProjectorByBin>;
 %include "stir/recon_buildblock/BackProjectorByBinUsingProjMatrixByBin.h"
+
+%include "stir/recon_buildblock/QuadraticPrior.h"
```
Comment (KrisThielemans) [inline src/swig/stir.i]: already at https://github.com/UCL/STIR/blob/081100dac83327ed231abad38dbf090998d87523/src/swig/stir.i#L1551

File: src/swig/stir.i [hunk 2]
```diff
@@ -1668,3 +1668,11 @@ namespace stir {
...
+#define elemT float
+%template (QuadraticPrior3DFloat) stir::QuadraticPrior<elemT >;
+
+%include "stir/recon_buildblock/PLSPrior.h"
```
Comment (KrisThielemans) [inline src/swig/stir.i]: already at https://github.com/UCL/STIR/blob/081100dac83327ed231abad38dbf090998d87523/src/swig/stir.i#L1552

File: src/swig/stir.i [hunk 3]
```diff
@@ -1668,3 +1668,11 @@ namespace stir {
   stir::RegisteredParsingObject<stir::BackProjectorByBinUsingProjMatrixByBin,
      stir::BackProjectorByBin>;
 %include "stir/recon_buildblock/BackProjectorByBinUsingProjMatrixByBin.h"
+
+%include "stir/recon_buildblock/QuadraticPrior.h"
+#define elemT float
+%template (QuadraticPrior3DFloat) stir::QuadraticPrior<elemT >;
```
Comment (KrisThielemans) [inline src/swig/stir.i]: already at https://github.com/UCL/STIR/blob/081100dac83327ed231abad38dbf090998d87523/src/swig/stir.i#L1597

File: src/swig/stir.i [hunk 4]
```diff
@@ -1668,3 +1668,11 @@ namespace stir {
...
+
+%include "stir/recon_buildblock/PLSPrior.h"
+#define elemT float
+%template (PLSPrior3DFloat) stir::PLSPrior<elemT >;
```
Comment (KrisThielemans) [inline src/swig/stir.i]: already at https://github.com/UCL/STIR/blob/081100dac83327ed231abad38dbf090998d87523/src/swig/stir.i#L1602

**Comment by robbietuk** [comment]
I think I can close this pull request as it does not seem to add anything

**Pull request closed without merging**